packages/components/src/ButtonsGroup/ButtonsGroup.spec.js

```diff
@@ -0,0 +1,44 @@
+import { render } from '@testing-library/react';
+import { ThemeProvider, createTheme } from '@platzily-ui/styling';
+import ButtonsGroup from './ButtonsGroup';
+
+describe('@Components/ButtonsGroup', () => {
+  it('Given the ButtonsGroup Component, when the props provide borderWrapperComponent with a number then the ButtonsGroup component will take border-radius attribute', () => {
```

Comment:
Lets add more tests here, we could test that for example the buttons rendered are the same length as the actions. And the last clicked button has the selected styles.